nibabel/filebasedimages.py

```diff
@@ -212,16 +211,16 @@ def __getitem__(self):
         '''
         raise TypeError("Cannot slice image objects.")
 
+    @deprecate_with_version('get_header method is deprecated.\n'
+                            'Please use the ``img.header`` property '
+                            'instead.',
+                            '2.1', '4.0')
```

Comment:
Yes, I was assuming that no-one was reading the docs, and so dated the deprecation from the first warning.